### PR TITLE
Landing page revamp and basic color scheme changes 

### PR DESCRIPTION
**Pull request by sea-lene**
## READ BELOW IF YOU'RE PARTAKING IN LANDING PAGE V0 REVAMP

_Written by Pan_

Since we have a multiple people, and different people can work on different small/big things, the least friction path forward may be to all work on the same branch! (s/o @sea-lene  for proposition)

Everyone can thus continue working on what Selene has started in this PR, and push commits to it. By running it from her branch, you'll see that some of the colors have already changed. However the code itself may not be clean and not follow best practices, I encourage refactoring of existing changes that she has made. 

@ICEPrey has mentioned that some changes currently there does not follow correct Shadcn practices. We should beware of that and fix it.

What we should implement now is [landing page V7](https://www.figma.com/design/yVJjhUlhUwwEDBul42yvB4/PearAI-stuff?node-id=0-1&t=obKguksqkyBH087v-1) tile within the Figma. Please refer to it and lmk of any questions!

The minimum required to merge this branch is to get the color theming down.

This is what currently looks like (June 17th). As you can see, e.g. the button font is still black instead of white. Need to make these changes (follow the [Figma](https://www.figma.com/design/yVJjhUlhUwwEDBul42yvB4/PearAI-stuff?node-id=0-1&t=obKguksqkyBH087v-1))
image

- @Fryingpannn

### CODE REVIEW

**Comment by Fryingpannn** [comment]
Yooo can we get some screenshots? 👀

**Comment by Fryingpannn** [comment]
Button font color on the green button should be light just like the black button
image

Missing buttons
Screenshot 2024-06-14 at 12 20 23 AM

File: tailwind.config.js [hunk 1]
```diff
@@ -86,14 +130,14 @@ module.exports = {
...
         foreground: "hsl(var(--foreground))",
-        primary: {
-          DEFAULT: "hsl(var(--primary))",
-          foreground: "hsl(var(--primary-foreground))",
-        },
-        secondary: {
-          DEFAULT: "hsl(var(--secondary))",
-          foreground: "hsl(var(--secondary-foreground))",
-        },
+        // primary: {
```
Comment (Fryingpannn) [inline tailwind.config.js:133]: Can delete these instead of comment

File: components/try.tsx [hunk 2]
```diff
@@ -19,10 +19,10 @@ export default function TryPear() {
             <h1 className="h1 mb-4" data-aos="fade-up">Try Pear.AI For Free</h1>
```
Comment (Fryingpannn) [inline components/try.tsx:19]:
```suggestion
            <h1 className="h1 mb-4" data-aos="fade-up">Try PearAI For Free</h1>
```

**Review by Fryingpannn — APPROVED** [review]

**Review by Fryingpannn — COMMENTED** [review]
Also let's change the black buttons to a light shade for now, any lighter shade or even transparent. The black is definitely way too strong for a secondary button when looking at the preview site

File: app/layout.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body
-        className={`${inter.variable} ${architects_daughter.variable} font-inter antialiased bg-gray-900 text-gray-200 tracking-tight`}
+        className={`${inter.variable} ${architects_daughter.variable} font-inter antialiased bg-white-100 text-secondary-main tracking-tight`}
```
Comment (Fryingpannn) [inline app/layout.tsx:28]: Are we going with background full white? I thought we're going just slightly beige
Screenshot 2024-06-14 at 12 29 51 AM

**Review by Dmedina20 — APPROVED** [review]
LGTM! Nice work!

**Review by Fryingpannn — APPROVED** [review]

**Comment by Fryingpannn** [comment]
SHOUTOUT TO THE LANDING PAGE ALPHA SQUAD FOR COMPLETING THIS QUEST!!!! @Ellahinator @somwrks @sea-lene @Alexx1105 @ICEPrey @jspEclipse @dscreet @shrix1 !!!!!

**Pull request merged**